src/components/CalloutCard/tests/CalloutCard.test.tsx

```diff
@@ -19,7 +19,12 @@ describe('<CalloutCard />', () => {
   );
 
   it('renders its children', () => {
-    expect(calloutCard.contains('<p>Content</p>'));
+    expect(
+      calloutCard
+        .find('p')
+        .first()
+        .contains('Content'),
+    ).toBe(true);
```

Comment:
I guess we can't assert `toContain` here?

Reply:
https://github.com/Shopify/web-foundation/blob/master/Best%20practices/Enzyme.md#expectations

Reply:
We're not currently using jest-enzyme so `toContain` doesn't exist.

@kvendrik raised #529 that says we should add it and I agree with you both. toContain() is a much nicer approach but lets add jest-enzyme and updating tests to use its new functionality in a separate PR.

Reply:
@BPScott [`toContain`](https://jestjs.io/docs/en/expect#tocontainitem) is part of Jest so you should be able to use it here 😄.

Reply:
oh hang on yes, I'm talking garbage sorry.

I did try doing this with toContain and it was failing which was rather odd :/

Reply:
It does work with strings though so this should work:
```tsx
expect(calloutCard.find('p').first()).toContain('Content');
```

Reply:
No need for `toContain`

```ts
expect(calloutCard.find('p').first().text()).toBe('Content');
```